src/pages/patientView/sampleManager.tsx

```diff
@@ -9,6 +9,11 @@ import DefaultTooltip from 'shared/components/DefaultTooltip';
 import {cleanAndDerive} from './clinicalInformation/lib/clinicalAttributesUtil.js';
 import styles from './patientHeader/style/clinicalAttributes.scss';
 
+import {getSpans} from './clinicalInformation/lib/clinicalAttributesUtil.js';
+import { PatientViewPageStore } from './clinicalInformation/PatientViewPageStore';
+
+const patientViewPageStore = new PatientViewPageStore();
```

Comment:
we don't want to instantiate a new store inside this component.  The store instance should be managed by the page.  Presentational components should define handlers (callbacks) that allow access to the store.  See handleSampleClick above.   Since we're dealing with this legacy code and have to hack, what I would do is use jQuery to bind to clicks on the DOM and then call the handler.   you can email me at lisman.aaron@gmail.com for more advice!  thanks for your work.